### PR TITLE
Enable unit tests

### PR DESCRIPTION
**Pull request by ghost**
### Description
Reenabled unit tests by regenerating abstractions packages
 
### Check List
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: build/scripts/packages.lock.json [hunk 1]
```diff
@@ -98,7 +98,7 @@
         "type": "Direct",
         "requested": "[0.1.0-canary.0.17, )",
         "resolved": "0.1.0-canary.0.17",
-        "contentHash": "n5aybS3KBbl6IQENfYEydRRGaDzu1lc7QZCdMNZGZ8Zu8VcoA+jc+3mhULzHRwwk8Y7XgQgC7OX1lOuRcAotHw==",
+        "contentHash": "9MsxDXQoSmbKxahAS1405BwUhVRdV+ZFDKULYe8wq9BnwNJrQOnZjgbRRhSK10q9B+6oy2UwSYzoA0bcpab+Cw==",
```
Comment (raymond-lum) [inline build/scripts/packages.lock.json:101]: Why does this change?

**Review by raymond-lum — COMMENTED** [review]
Do you know why there are still 6 unit test failures in the pipeline if this is supposed to fix them all to 100%?

**Comment by ghost** [comment]
Closing this PR as it is not required to regenerate packages. Will create a cleaner PR

**Review by Yury-Fridlyand — APPROVED** [review]

**Comment by ghost** [comment]
> Closing this PR as it is not required to regenerate packages. Will create a cleaner PR

Reopening as the assumption was wrong due to different behaviours of local environment and GitHub Actions.

File: build/scripts/packages.lock.json (hunk 1 above)
Comment (ghost) [reply build/scripts/packages.lock.json:101]: I had to regenerate to change versions from 7.0.0 to 1.0.0

**Review by alexmeizer — APPROVED** [review]

**Pull request merged**